CRM/Contribute/Page/ContributionPage.php

```diff
@@ -299,15 +299,11 @@ public function run() {
       return $controller->run();
     }
     elseif ($action & CRM_Core_Action::UPDATE) {
-      $config = CRM_Core_Config::singleton();
-
       // assign vars to templates
       $this->assign('id', $id);
       $this->assign('title', CRM_Core_DAO::getFieldValue('CRM_Contribute_DAO_ContributionPage', $id, 'title'));
       $this->assign('is_active', CRM_Core_DAO::getFieldValue('CRM_Contribute_DAO_ContributionPage', $id, 'is_active'));
-      if (in_array('CiviMember', $config->enableComponents)) {
-        $this->assign('CiviMember', TRUE);
-      }
+      $this->assign('CiviMember', CRM_Core_Component::isEnabled('CiviMember'));
```

Comment:
Unconditional assign to align with our efforts to reduce undefined template vars.